windows/README.md

```diff
@@ -98,6 +98,42 @@ the ```driver/src/winmonitor_gen.c``` file.
 Finally, rebuild the driver. Note that new OS versions may considerably change the kernel structure, requiring upgrading
 the generation scripts.
 
+Code Coverage for Windows Binaries
+==================================
+
+Binaries produced by Microsoft tools contain line information in PDB files. These files have a proprietary
+format and are not readable by Linux tools. This poses several challenges in order to covert program counters
+produced by S2E's coverage plugin into say LCOV reports.
+
+The following instructions explain how to get LCOV coverage reports for Windows binaries.
+
+- Build the solution in Visual Studio
+
+- Run ```pdbparser.exe``` as follows:
+
+    ```
+    ./x64/Release/pdbparser.exe -l my_binary.exe my_binary.pdb > my_binary.exe.lines
+    ```
+
+    The ```*.lines``` file contains line information in JSON format.
+
+- Run your binary in S2E using ```s2e-env```. At the end of the run, you should have ```tb-coverage*.json```
+  files in your ```s2e-last``` folder.
+
+- Run ```s2e coverage lcov my_binary```. Make sure that the ```my_binary.exe.lines``` is located in the same
+  directory as ```my_binary.exe```.  This should produce the ```coverage.info``` file, which contains
+  LCOV coverage info.
+
+- You must use LCOV on Windows in order to generate the report, because the LCOV files contain Windows path.
+  You can also patch the files yourself to convert the paths inside to the Linux format. Run the following command in MSYS after
+  installing LCOV from ```https://github.com/linux-test-project/lcov```:
```

Comment:
Can you make this an actual link, rather than formatted as "code". I'm pretty sure just having the URL without the ` ` `around it will make it clickable.